.github/workflows/main.yml

```diff
@@ -6,8 +6,8 @@ jobs:
       fail-fast: false
       matrix:
         gemfile: [rails_6_1, rails_7_0, rails_7_1, rails_master]
-        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
-        ruby: ['3.0', 3.1, 3.2, 3.3]
+        # quote version numbers, they are not "numbers"
+        ruby: ['3.1', '3.2', '3.3']
```

Comment:
I thought about keeping the original comment saying _remember to quote for .0 numbers_, but I think its just easier to quote _all_ the numbers and not have one special case.

Reply:
Good call